myosuite/utils/__init__.py

```diff
@@ -16,4 +17,8 @@ def import_gym():
     else:
         raise ModuleNotFoundError(help)
     return gg
-gym = import_gym()
\ No newline at end of file
+gym = import_gym()
+
+__all__ = [
```

Comment:
What is this for?

Reply:
this is removed, all is integrated with current sb3 implementation